src/Saunter/Saunter.csproj

```diff
@@ -47,7 +47,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="3.0.0" />
-    <PackageReference Include="Namotion.Reflection" Version="1.0.23" />
+    <PackageReference Include="Namotion.Reflection" Version="2.0.1" />
```

Comment:
Updated to latest Namotion.Reflection - this was my main purpose of this PR initially ;-)

Reply:
maybe remove the reference to Namotion.Reflection as it should be brought in by njsonschema?

Reply:
Good point. I thought in the past you needed a direct reference if you used types directly. Not the case anymore... Removed.